packages/react-drag-drop/src/next/components/DragDrop/Draggable.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/DragDrop/drag-drop';
+import { DragButton } from './DragButton';
+
+export interface DraggableProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside DragDrop */
+  children?: React.ReactNode;
+  /** Don't wrap the component in a div. Requires passing a single child. */
```

Comment:
Maybe "Flag indicating that a div should not wrap the componet". Or something along those lines.